pkg/zfs/cmd/v1alpha1/zfs/jsonproperty/builder.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vjsonprop
```

Comment:
I like the approach having variable for the command, but it will be better if we make this package specific to `listsnap` command only. We can surely use this for other command, but if arguments are more and different per command then it will be difficult to maintain this code. So better to rename this package as `listsnap` and make the command internal to package. What are your thoughts on this?

Reply:
I observed JSON commands in ZFS are with a single property i.e `zfs/zpool <property_name> <dataset>` so went with this genaric solution.

Reply:
But the concern of maintaining also valid so making it listsnap and moving code accordingly.